src/pages/DateTimeFormat/index.js

```diff
@@ -627,7 +627,7 @@ const Home = () => {
               renderItem={item => (
                 <List.Item>
                   <List.Item.Meta
-                    avatar={<Avatar src={item.avatar} />}
+                    avatar={<Avatar src={item.avatar} alt="avatar" />}
```

Comment:
may I suggest something like `alt={`${item.title} page avatar`}` or add an alt property on the item and use it there, if not all lines will have an alt of avatar and the user cannot distinguish between them, right?